frontend/workflows/redisexperimentation/src/tests/experiment-details.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import { FormFields } from "@clutch-sh/experimentation";
+import { ExperimentDetails } from "../start-experiment";
+
+jest.mock("react-router-dom", () => {
+  return {
+    ...jest.requireActual("react-router-dom"),
+    useNavigate: jest.fn(),
+  };
+});
+
+describe("Start Experiment workflow", () => {
```

Comment:
adding new snapshot tests to make sure that the fields in start redis experimentation screen look correctly.